app/workers/ckan_sync_worker.rb

```diff
@@ -0,0 +1,31 @@
+class CKANSyncWorker
+  def perform
+    actions = CKAN::V26::VersionDiff.new.call
+    create_new_datasets(actions[:create])
+    update_existing_datasets(actions[:update])
+    delete_old_datasets(actions[:delete])
+  end
+
+private
+
+  def create_new_datasets(packages)
+    packages.each do |package|
+      Dataset.create(title: "title",
```

Comment:
I wonder whether we should extract this into a separate job? If any of the datasets fail to create, update or delete then whole job will fail. If we break the work down into individual jobs the failure will only effect the isolated dataset and not block the whole batch.